test/features/random/enum.test.ts

```diff
@@ -96,15 +96,50 @@ describe('Random enum', () => {
     expect(thirdMock.direction).toBe(Direction.None);
   });
 
-  it('should have position of "computed" property, because transformer don\'t support computed Enum values', () => {
+  it('should get the correct const enum member computed property', () => {
```

Comment:
Do you think we should add a test outside the random functionality?

Reply:
done